prow/lib.sh

```diff
@@ -115,7 +115,7 @@ function build_images() {
   SELECT_TEST="${1}"
 
   # Build just the images needed for tests
-  targets="docker.pilot docker.proxyv2 "
+  targets="docker.pilot docker.proxyv2 docker.operator "
```

Comment:
I think this was temporary?

Reply:
yes once I see all the tests passing I will revert that and remove the do-not-merge